pkgs/os-specific/linux/systemd/default.nix

```
@@ -98,6 +98,7 @@
 , withImportd ? !stdenv.hostPlatform.isMusl
 , withKmod ? true
 , withLibBPF ? lib.versionAtLeast buildPackages.llvmPackages.clang.version "10.0"
+    && buildPackages.targetPackages.llvmPackages.compiler-rt.meta.available
```

In general we should try to avoid `meta.available`.  Please check that this suggestion matches your intention (`llvmPackages` is still a bit mysterious to me):

```suggestion
    && lib.meta.availableOn stdenv.targetPlatform llvmPackages.compiler-rt
```

We should try to avoid `meta.available` because it is affected by impure environment variables like `NIXPKGS_ALLOW_NONSOURCE`.  It's okay for environment variables like that to cause a build to fail, but if we start switching dependencies in and out based on `meta.available` we can get builds that succeed *differently* when you flip those on.  *This actually happened to me shortly after I started using `allowNonSource=false`.*

Despite the similar-sounding name, `lib.meta.availableOn` doesn't have this problem.

We should probably deprecate `meta.available`, or at least make it internal to `stdenv` where we can be sure that code using it will be careful to let it influence only the success/failure, but not the result, of a build.